Packs/Salesforce/Integrations/Salesforce/Salesforce_description.md

```diff
@@ -1,5 +1,5 @@
-To set up Salesforce to work with Cortex XSOAR:
+To set up Salesforce to work with Demisto:
 Add a new connected App in salesforce by following the instruction [here](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_defining_remote_access_applications.htm)
 If you already have a connected App, go to “Setup” -> “App Manager” -> choose the correct App from the list and press “View”.
 The Consumer Key / Secret is under “API (Enable OAuth Settings)”
-For detailed instructions see the Credentials walkthrough section at [Cortex XSOAR support](https://support.demisto.com/hc/en-us/articles/360001848133-Integration-Salesforce).
\ No newline at end of file
+For detailed instructions see the Credentials walkthrough section at [demisto support](https://support.demisto.com/hc/en-us/articles/360001848133-Integration-Salesforce).
```

Comment:
Again, why is Cortex XSOAR changed to demisto?  
Also, I was not able to access the URL.  Is it correct?